src/activitylog/activitylog.html

```diff
@@ -14,27 +18,38 @@ <h1>Extension Activity Monitor</h1>
       <div class="notice"></div>
       <div class="activity-log-wrapper">
         <h2>Activity Logs</h2>
-        <div class="activity-log-container">
-          <div class="log-table">
-            <table>
-              <thead>
-                <tr>
-                  <th>Extension Id</th>
-                  <th>Timestamp</th>
-                  <th>API Type</th>
-                  <th>API Name</th>
-                  <th>View Type</th>
-                </tr>
-              </thead>
-              <tbody></tbody>
-            </table>
-          </div>
-          <div class="show-log-detail-wrapper" hidden>
-            <div id="showLogDetails"></div>
-            <div class="close"></div>
-          </div>
-        </div>
+        <log-view></log-view>
       </div>
     </div>
+    <template class="log-table-template">
+      <link rel="stylesheet" href="/lib/web-component/log-view/log-view.css" />
+      <div class="activity-log-container">
+        <div class="log-table-wrapper">
+          <table>
+            <thead>
+              <th>Extension Id</th>
+              <th>Timestamp</th>
+              <th>API Type</th>
+              <th>API Name</th>
+              <th>View Type</th>
+            </thead>
+            <tbody></tbody>
+          </table>
+        </div>
+        <div class="log-detail-wrapper" hidden>
+          <div id="logDetails"></div>
```

Comment:
Use `class` instead of `id` when there may be more than one instance of an element. Anything in a custom element could in theory be used more than once, so use `class="logDetails"` instead.

Reply:
done :-)